docs/index.rst

```diff
@@ -16,10 +16,9 @@ Contents:
 
    introduction/index
    gettingstarted/index
-   indepth/index
-   advanced/index
+   using/index
+   concepts/index
    reference/index
-   gettinginvolved/index
-   roadmap/index
+   releasenotes/index
    faq/index
-   authors
+   gettinginvolved/index
```

Comment:
Add a newline at end of file. This makes this and future patches clearer with tools like `git diff`, and ensures files display correctly in terminals.